### PR TITLE
Update splash-screens-and-icons.md

### PR DESCRIPTION
**Pull request by bechtold**
Updated docs to reflect the current state of the assets tool.

### CODE REVIEW

**Review by dtarnawsky — APPROVED** [review]

**Review by markemer — APPROVED** [review]
LGTM

File: docs/main/guides/splash-screens-and-icons.md [hunk 1]
```diff
@@ -1,32 +1,51 @@
...
 
-Initial support for splash screen and icon generation is now available. For complete details, see the [cordova-res docs](https://github.com/ionic-team/cordova-res).
+This tool will crop and resize JPEG and PNG source images to generate icons and splash screens for iOS, Android, and Progressive Web Apps. For complete details, see the [capacitor assets docs](https://github.com/ionic-team/capacitor-assets).
 
-First, install `cordova-res`:
+First, install `capacitor-assets`:
 
 ```bash
-npm install -g cordova-res
+npm install @capacitor/assets
```
Comment (dtarnawsky) [inline docs/main/guides/splash-screens-and-icons.md:17]: I would modify this to: 
npm install --save-dev @capacitor/assets

File: docs/main/guides/splash-screens-and-icons.md [hunk 2]
```diff
@@ -1,32 +1,51 @@
...
 ```
 
-Next, run the following to generate all images then copy them into the native projects:
+Then, generate the assets and provide the background colors that will be used to generate background layers for icons:
 
-```bash
-cordova-res ios --skip-config --copy
-cordova-res android --skip-config --copy
+```shell
+npx capacitor-assets generate --iconBackgroundColor '#eeeeee' --iconBackgroundColorDark '#222222' --splashBackgroundColor '#eeeeee' --splashBackgroundColorDark '#111111'
```
Comment (dtarnawsky) [inline docs/main/guides/splash-screens-and-icons.md:33]: Change capacitor-assets to @capacitor/assets here

**Comment by dtarnawsky** [comment]
The Capacitor docs have been updated for this topic in this [PR](https://github.com/ionic-team/capacitor-docs/pull/161) and they include the change from `cordova-res` to `@capacitor/assets`. The new page is [here](https://capacitorjs.com/docs/next/guides/splash-screens-and-icons).

Thank you for your PR @bechtold

**Pull request closed without merging**